manifests/platform.yaml

```diff
@@ -107,6 +107,10 @@ spec:
           profile: dryrun
         volumes:
           profile: dryrun
+  - name: cray-kyverno
+    source: csm-algol60
+    version: v2.3.0
```

Comment:
In the version field prefix "v" is required? For other charts its not mentioned. This might cause challenges in version upgrade checks in the PR.

Reply:
This comes from upstream so thought of keeping the same to avoid confusion.